api/src/main/java/com/codeforcommunity/dto/ApiDto.java

```diff
@@ -102,4 +103,19 @@ protected boolean isEmpty(String str) {
   protected boolean passwordInvalid(String pass) {
     return pass == null || pass.trim().length() < 8;
   }
+
+  /**
+   * Checks to see if the a privilege level is in the enum.
+   *
+   * @param level the privilege level to check
+   * @return a boolean representing whether the given privilege level is invalid or not.
+   */
+  protected boolean privilegeLevelInvalid(String level) {
+    for (PrivilegeLevel val : PrivilegeLevel.values()) {
+      if (val.name().equals(level)) {
```

Comment:
Maybe do a .toUpperCase() or .toLowerCase() here just so no one ends up with weird capitalization errors

Reply:
Ah I just realized superAdmin is camel case.... We probably want to change that to not depend on capitalization

Reply:
On the front end a drop down is used so capitalization should not be an issue since there's preset options

Reply:
Made the change anyway since it shouldn't really matter